spec/system/reassign_an_application_confirmation_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe 'Reassigning an application confirmation errors' do
+  let(:crime_application_id) { '696dd4fd-b619-4637-ab42-a5f4565bcf4a' }
+
+  let(:confirm_path) do
+    new_crime_application_reassign_path(crime_application_id)
+  end
+
+  let(:assignee) do
+    User.create(
+      first_name: 'Fred',
+      last_name: 'Smitheg',
+      auth_oid: SecureRandom.uuid,
+      email: 'Fred.Smitheg@justice.gov.uk'
+    )
+  end
+
+  before do
+    Assigning::AssignToUser.new(
+      user_id: assignee.id,
+      to_whom_id: assignee.id,
+      assignment_id: crime_application_id
+    ).call
+
+    visit '/'
+    click_on 'All open applications'
+    click_on('Kit Pound')
+    click_on('Reassign to myself')
+  end
+
+  describe 'reloading the confirm page after a asignment has been unassigned' do
```

Comment:
needs an another 's' in assignment here 👍

Reply:
done